### ISSUE TITLE
Doesn't work on (High) Sierra

### ISSUE THREAD

**Issue by ckotte** [MatthewWest/pass-alfred#11]
I also get a message that the workflow contains a python library that is incompatible with macOS Sierra 10.12.4+: [Incompatible Python workflow library](https://www.alfredapp.com/help/troubleshooting/incompatible-python-workflow-library/)

I can only execute the pass workflow after executing the fixum workflow. The workflows fails with:

```
[2018-04-23 18:20:45][input.scriptfilter] Queuing argument 'a'
[2018-04-23 18:20:45][input.scriptfilter] Script with argument '(null)' finished
[2018-04-23 18:20:45][ERROR: input.scriptfilter] Code 1: 18:20:45 workflow.py:2249 DEBUG    ---------- pass ----------
18:20:45 workflow.py:1657 DEBUG    reading settings from /Users/xxx/Library/Application Support/Alfred 3/Workflow Data/org.passwordstore.pass/settings.json
18:20:45 workflow.py:2266 ERROR    'ascii' codec can't decode byte 0xc2 in position 3: ordinal not in range(128)
Traceback (most recent call last):
  File "/Users/xxx/.backup-mac/Alfred.alfredpreferences/workflows/user.workflow.C882CE17-5AEA-4CAF-B448-480D4575EC3E/workflow/workflow.py", line 2259, in run
    func(self)
  File "pass.py", line 143, in main
    results = wflow.filter(query, rel_paths, key=search_key_for_pw, min_score=20)
  File "/Users/xxx/.backup-mac/Alfred.alfredpreferences/workflows/user.workflow.C882CE17-5AEA-4CAF-B448-480D4575EC3E/workflow/workflow.py", line 2074, in filter
    value = key(item).strip()
  File "pass.py", line 19, in search_key_for_pw
    return u' '.join(elements)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 3: ordinal not in range(128)
18:20:45 workflow.py:2289 DEBUG    ---------- finished in 0.018s ----------
[2018-04-23 18:20:45][input.scriptfilter] <?xml version="1.0" encoding="utf-8"?>
<items><item valid="no"><title>Error in workflow 'pass'</title><subtitle>'ascii' codec can't decode byte 0xc2 in position 3: ordinal not in range(128)</subtitle><icon>/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertStopIcon.icns</icon></item></items>
```

**Comment by MatthewWest** [comment]
I'm not currently using the pass Alfred workflow, so I probably won't get around to debugging & fixing this. I'd welcome a pull request to fix it though!

**Comment by ckotte** [comment]
Is fixed with #12.
